GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1438,13 +1436,6 @@ private void OnGridViewSelectionChanged(object sender, EventArgs e)
             if (_gridView.SelectedRows.Count > 0)
             {
                 _latestSelectedRowIndex = _gridView.SelectedRows[0].Index;
-
-                // if there was selected a new revision while data is being loaded
-                // then don't change the new selection when restoring selected revisions after data is loaded
-                if (_isRefreshingRevisions && !_gridView.UpdatingVisibleRows)
-                {
-                    _selectedObjectIds = _gridView.SelectedObjectIds;
-                }
```

Comment:
This might be a regression for some users - but acceptable IMO.

Reply:
If you make a selection while loading, the selection is used.
This code was not used.